write_column.go

```diff
@@ -14,20 +15,13 @@ type ColumnarStatement interface {
 }
 
 func (stmt *stmt) ColumnWriter() ColumnWriter {
-	stmt.ch.data.reserveColumns()
-	return stmt.ch.data
+	stmt.ch.block.Reserve()
+	return stmt.ch.block
 }
 
 func (stmt *stmt) ColumnWriterEnd(rows uint64) error {
-	writtenBlocks := stmt.counter / stmt.ch.blockSize
-	stmt.counter += int(rows)
-	stmt.ch.data.numRows += rows
-
-	var err error
-	if stmt.counter/stmt.ch.blockSize > writtenBlocks {
-		err = stmt.ch.data.write(stmt.ch.serverRevision, stmt.ch.conn)
-	}
-	return err
```

Comment:
Is there a way how to force chunking of input into blocks? The `blockSize` put an upper bound on how much memory would data would be in buffers before sending it to server. Is there a way to control when a block should be written and its structure recycled? Something like:

```
stmt.ColumnWriterEnd(1)
if stmt.NumRows > 100000 {
    stmt.WriteAndReset()
}
```